tests/test_superform.py

```diff
@@ -64,4 +80,43 @@ def test_validation(self):
         self.assertEqual(form.is_valid(), False)
         self.assertTrue(form.errors['username'])
         self.assertTrue(form.errors['emails'])
-        self.assertIsInstance(form.errors['emails'], ErrorList)
+        self.assertIsInstance(form.errors['emails'], list)
```

Comment:
If the error is a `list` and no `ErrorList`, then the HTML output for the errors will break. The error list takes care of the formatting like
```
<ul class="errorlist"><li><ul class="errorlist"><li>email<ul class="errorlist"><li>This field is required.</li></ul></li></ul></li></ul>
```

The `list` instance would only display `[{'email': [u'This field is required.']}]`